ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/validator/preexecution/PreExecutingValidatingCommitter.scala

```diff
@@ -59,24 +56,22 @@ class PreExecutingValidatingCommitter[StateValue, ReadSet, WriteSet](
   /** Pre-executes and then commits a submission.
     */
   def commit(
+      participantId: ParticipantId,
```

Comment:
There are different types of participants for a transaction, hence, I'd suggest keeping the original argument naming here.

Reply:
Done.